README.md

```diff
@@ -36,13 +50,13 @@ We're excited to have you on board! Here's how you can contribute:
 
 To stay updated and listen to our shows, follow the hosts for the latest show links. Your participation and support make Tech Commute a better community!
 
-[Jason Torres](https://twitter.com/TasonJorres) **&** [Jacob Ashley](https://twitter.com/arcadejacob)
+[Jason Torres](https://twitter.com/TasonJorres) **&** [Jacob Ashley](https://twitter.com/arcadejacob) **&** [Ellie Zubrowski](https://x.com/elliezub)
```

Comment:
These probably redirect, but maybe we want to change the URLs for Jason and Jacob to 'https://x.com...'?

Reply:
> These probably redirect, but maybe we want to change the URLs for Jason and Jacob to 'https://x.com...'?

Updated the links! ^^